examples/design/simple_reactor.ex.py

```diff
@@ -505,15 +506,16 @@ def build_xyz(self) -> PhysicalComponent:
         },
     },
     "TF Coil": {
-        "Designer": {
+        "params": {},
```

Comment:
Just double-checking - does this _need_ to be here, or is the params key still optional?

Reply:
No it doesn't. The params key is optional, so if some key doesn't have "params" uses an empty dict.
If you try to get params for something (say "TF Coil", "designer" in this), it will warn you if the final dict is empty.